openssl-sys/build/find_normal.rs

```diff
@@ -14,7 +14,15 @@ pub fn get_openssl(target: &str) -> (PathBuf, PathBuf) {
         (lib_dir, include_dir) => {
             let openssl_dir = env("OPENSSL_DIR").unwrap_or_else(|| find_openssl_dir(target));
             let openssl_dir = Path::new(&openssl_dir);
-            let lib_dir = lib_dir.unwrap_or_else(|| openssl_dir.join("lib"));
+            let lib_dir = lib_dir.unwrap_or_else(|| {
+                // OpenSSL 3.0 now puts it's libraries in lib64/ by default.
+                // Check if that exists and if not, use lib/.
+                if openssl_dir.join("lib64").exists() {
```

Comment:
Is it possible that user installs OpenSSL to `/usr` and OpenSSL libs reside in `/usr/lib` but `/usr/lib64` directory also exists?

```bash
# ls /usr/lib64
ld-linux-x86-64.so.2  ld-lsb-x86-64.so.2  ld-lsb-x86-64.so.3
```

Reply:
Hmm, yes it is. In the `/usr` case I'd kind of expect it to be covered by `pkg-config`. What do you think @sfackler ?

Reply:
pkg-config would cover a /usr install in normal contexts, but not if `OPENSSL_DIR=/usr` was set explicitly. We could just add both `$OPENSSL_DIR/lib` and `$OPENSSL_DIR/lib64` as search paths?

Reply:
Sure, I'll do that.

Reply:
Done.